datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/ci.py

```diff
@@ -210,7 +210,7 @@ def validate_master_jobs(fix, repo_data, testable_checks, cached_display_names):
                 display_queue.append((echo_failure, message))
 
     if not success:
-        message = 'Try running `ddev validate ci --fix`'
+        message = 'Try running `ddev validate ci --sync`'
```

Comment:
I still see `fix` 🤔  

https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/ci.py#L499-L501

Reply:
Oh I got it, in `ddev` it's `sync` and it's `fix` in `datadog_checks_dev`. I think it's a bit confusing, could you add a comment in the code about that?

Reply:
I added a comment linking to the new param. Let me know if that is descriptive enough. I couldn't find any good documentation on how to create the alias.

Reply:
Thanks!